Library/ENV/4.3/cc

```diff
@@ -1,6 +1,7 @@
-#!/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby -W0
+#!/usr/bin/ruby -W0
 
-$:.unshift Dir["/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/lib/ruby/{1.8,2.0.0}"].first
+lpath = Dir["/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/lib/ruby/{1.8,2.0.0}"].first
+$:.unshift lpath unless lpath.nil?
```

Comment:
Yes. Without this change it would try to prepend nil to $: which in turn broke ```require``` directives.